layouts/partials/navigation.html

```diff
@@ -1,13 +1,13 @@
 {{ $appliedScopes := .Site.Params.scopes }}
 {{ $scopingEnabled := .Site.Params.scopesEnabled }}
-{{ $rootUrl := ( relURL "/")}}
+{{ $rootUrl := strings.TrimSuffix "/" ( relURL "/")}}
 
 <nav class="navbar scrollable-container">
     <div class="navbar-header">
-        <a href="{{ .Site.BaseURL }}" class="brand">
-            {{ $navImg := (printf "%s%s" .Site.BaseURL "images/logo.png") }}
+        <a href="{{ $rootUrl }}" class="brand">
+            {{ $navImg := (printf "%s%s" $rootUrl "images/logo.png") }}
```

Comment:
As posted in Slack, I had a problem with this not pulling through. I traced this to `baseURL` not being pulled through in my content-builder. This could be an error on my side but I used this opportunity to upskill on Hugo and we might not be using `relURL` correctly.

My broken image with this code:

<img width="278" alt="Screenshot 2022-12-20 at 17 32 20" src="https://user-images.githubusercontent.com/1186788/208704753-eeff5d5d-2267-4103-bbfa-ddd4a1793716.png">

If I change this line to pipe into `relURL` (see documentation link below), I then get the image:

```go
{{ $navImg := "images/logo.png" | relURL }}
# returns:  /docs/presidium-docs-internal/images/logo.png
```

<img width="409" alt="image" src="https://user-images.githubusercontent.com/1186788/208707388-d24b3a3d-9890-4ffb-a86f-021b514e744f.png">



* Hugo doc ref for use of pipe: https://gohugobrasil.netlify.app/functions/relurl/
* We are using the pipe into `relURL` in only one place https://github.com/SPANDigital/presidium-theme-website/blob/83fc6ad9c5a6957bc376b8e156b2d0a3d3576a92/layouts/partials/json-navigation.html#L16

Reply:
Thanks! `relURL` works pretty well but I found one issue with it. If the image path is absolute e.g `/images/logo.png` then it outputs an absolute path: `/images/logo.png`. So we have to handle the edge case where the user might add a leading slash to the path. 

The best solution I found was to get the path from the baseUrl and join it with the image path
Eg. if baseUrl is `localhost:9000/docs/span-handbook/` & logo is `/images/image.png`
```
{{ $url := urls.Parse .Site.BaseURL }} output => `/docs/span-handbook/`
{{ $rootUrl := path.Clean $url.Path }} output => `/docs/span-handbook`
{{ $navImg = path.Join $rootUrl .Site.Params.logo }} output  => `/docs/span-handbook/images/image.png`
```

Reply:
The `path.Clean` is used because Hugo appends extra slashes when you set the path dynamically. Like we do with the WYSIWYG builder:
```
hugo serve --baseUrl /docs/span-handbook
```

The `$url` will be `///docs/span-handbook/`